### ISSUE TITLE
How to finetune the face model

### ISSUE THREAD

**Issue by mxmxlwlw** [happynear/NormFace#24]
Hi,
   I've trained a model using ms-celeba dataset. And I want to finetune it using VGGFace2. I use centerloss. But when I finetune it, the softmax loss decrease to 0.7 very quickly. However it increase to 4-5 after that. So, my question is how to finetune a face recognition when using centerloss. Thank you!

**Comment by mxmxlwlw** [comment]
Hi,
     I made a mistake. Not the loss's fault... Thank you any way.

**Comment by FredaZhang338** [comment]
@mxmxlwlw 
hello,can you share some information about the instructions of training a model based on the normface?

**Comment by mxmxlwlw** [comment]
@ManningZhang
I can't, I haven't try it yet.
